src/libraries/System.Private.CoreLib/src/System/Type.cs

```diff
@@ -149,6 +149,8 @@ public virtual Type[] GetGenericParameterConstraints()
         protected abstract ConstructorInfo? GetConstructorImpl(BindingFlags bindingAttr, Binder? binder, CallingConventions callConvention, Type[] types, ParameterModifier[]? modifiers);
 
         [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern",
+            Justification = "Linker doesn't recongnize GetConstructors(BindingFlags.Public) but this is safe")]
```

Comment:
Please don't use "safe"... it resembles security, which this is not about (I know we use it a lot, but we should try to stay away from it)